source/NPC.cpp

```diff
@@ -675,6 +791,39 @@ NPC NPC::Instantiate(const PlayerInfo &player, map<string, string> &subs, const
 		result.system = location.PickSystem(origin);
 	if(!result.system)
 		result.system = (isAtDestination && destination) ? destination : origin;
```

Comment:
Warp figured out that #11506 was caused by this line. `destination` used to be the system pointer parameter to this function. There is now a `destination` field on the class that is always null when Instantiate is called. The class variable having the same name allowed this to compile even though its meaning changed.

It seems like `result.destination` will always be non-null, so we can also simplify this while we're here.
```suggestion
		result.system = isAtDestination ? result.destination : origin;
```